pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -941,6 +943,38 @@ func (o *MustGatherOptions) newPod(node, image string, hasMaster bool) *corev1.P
 
 	cleanedSourceDir := path.Clean(o.SourceDir)
 	volumeUsageChecker := fmt.Sprintf(volumeUsageCheckerScript, cleanedSourceDir, cleanedSourceDir, o.VolumePercentage, o.VolumePercentage, executedCommand)
+	//fmt.Sprintf(volumeUsageCheckerScript,cleanedSourceDir,o.VolumePercentage,executedCommand,)
+	
+	excludedTaints := []corev1.Taint{
+		{Key: unreachableTaintKey, Effect: corev1.TaintEffectNoExecute},
+		{Key: unreachableTaintKey, Effect: corev1.TaintEffectNoSchedule},
+	}
+
+	var tolerations []corev1.Toleration
+	if node == "" && hasMaster {
+		tolerations = append(tolerations, corev1.Toleration{
+			Key:      "node-role.kubernetes.io/master",
+			Operator: corev1.TolerationOpExists,
+			Effect:   corev1.TaintEffectNoSchedule,
+		})
+	}
+
+	tolerations = append(tolerations, corev1.Toleration{
```

Comment:
You could replace the first `var tolerations []corev1.Toleration` with:
```go
tolerations := []corev1.Toleration{
	{
		Key:      "node.kubernetes.io/not-ready",
		Operator: corev1.TolerationOpExists,
		Effect:   corev1.TaintEffectNoSchedule,
	},
}
```

directly.